protobuf/proto/agent.proto

```diff
@@ -67,6 +67,7 @@ message VersionDeployRequest {
 message DeployRequest {
 
   string id = 1;
+  string name = 2;
```

Comment:
name of the container?

Reply:
yes

Reply:
could we please use containerName instead?

Reply:
yes